{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:freeopcua:opcua-asyncio:*:*:*:*:*:python:*:*"
        ],
        "fix_commit": "https://github.com/FreeOpcUa/opcua-asyncio/commit/f6603daa34a93a658f0e176cb0b9ee5a6643b262",
        "version_end_excluding": {
            "freeopcua:opcua-asyncio": "0.9.96"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "PYTHON",
    "licences": {
        "database": "",
        "affected_repository": "GNU Lesser General Public License v3.0"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2023-26151"
        ],
        "CWE": [
            "CWE-400"
        ]
    }
}